tests/baselines/reference/classExpression4.types

```diff
@@ -1,22 +1,22 @@
 === tests/cases/conformance/classes/classExpressions/classExpression4.ts ===
 let C = class {
->C : typeof (Anonymous class)
+>C : any

```

Comment:
@ahejlsberg @mhegazy is this correct? Or should we do something special for this case